### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,10 +25,36 @@ PK-Sim® uses building blocks that are grouped into [**Individuals**](https://do
...
+
+Validation and automation tools include for example:
+
+* **Installation Validator**: enables "1-Click" validation of the OSP Suite installation on a target computer. The validation is performed by execution of the predefined set of simulation scenarios and comparison of the simulated results with the (validated) reference values.
+* **Command Line Interface** (*CLI*): allows batch processing of multiple projects in PK-Sim and is described in [Command Line Interface - CLI](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-command-line-interface).
+
+### Toolboxes
+
+The MoBi® toolbox for R is an interface to the common statistical computing environment. Basically, the R-toolbox can be used to access and modify model parameters as well as to execute simulations and retrieve results. That way, the R-toolbox can be used to script or code batch simulations, analysis tasks, or customized workflows to any complexity. Results can be visualized using the options available in the respective environment. The MoBi® toolbox for R is described in detail in [R documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction).
```
Comment (msevestre) [inline README.md:46]: Let's not call it the mobi tool box. It has nothing to do with Mobi anymore

File: README.md [hunk 2]
```diff
@@ -30,10 +25,36 @@ PK-Sim® uses building blocks that are grouped into [**Individuals**](https://do
...
+
+**OSP Qualification Framework and MoBi® toolbox for R are not included into the main OSP Suite setup and must be installed separately. Installation instructions are provided in the documentation of the tools.**
```
Comment (msevestre) [inline README.md:48]: Same here

File: README.md (hunk 1 above)
Comment (Yuri05) [reply README.md:46]: OSP Toolbox for R?

File: README.md [hunk 3]
```diff
@@ -43,9 +43,9 @@ Validation and automation tools include for example:
 
 ### Toolboxes
 
-The MoBi® toolbox for R is an interface to the common statistical computing environment. Basically, the R-toolbox can be used to access and modify model parameters as well as to execute simulations and retrieve results. That way, the R-toolbox can be used to script or code batch simulations, analysis tasks, or customized workflows to any complexity. Results can be visualized using the options available in the respective environment. The MoBi® toolbox for R is described in detail in [R documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction).
+The OSP toolbox for R is an interface to the common statistical computing environment. Basically, the R-toolbox can be used to access and modify model parameters as well as to execute simulations and retrieve results. That way, the R-toolbox can be used to script or code batch simulations, analysis tasks, or customized workflows to any complexity. Results can be visualized using the options available in the respective environment. The OSP toolbox for R is described in detail in [R documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction).
```
Comment (msevestre) [inline README.md:46]: Same line we call it
R-toolbox
and
OSP toolbox for R

Should we just call it R-toolbox?

**Review by msevestre — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yuri05) [reply README.md]: changed to OSP toolbox for R

File: README.md (hunk 3 above)
Comment (Yuri05) [reply README.md]: ok, renamed just to "R-toolbox" everywhere

**Comment by StephanSchaller** [comment]
just a (late) comment: if we talk about R packages, we normally only use the package name. So in our case we should just call it OSP-R package? The term "toolbox" was inherited from Matlab I would say and is not usually used for R.

**Comment by PavelBal** [comment]
I agree with Stephan, I would prefer not to use the word "toolbox".

**Comment by Yuri05** [comment]
@StephanSchaller @PavelBal ok... feel free to create a new PR :)
The V10 documentation should be adjusted accordingly.
* [here](https://docs.open-systems-pharmacology.org/v/v10/open-systems-pharmacology-suite/modules-philsophy-building-blocks#toolboxes)
* [here](https://docs.open-systems-pharmacology.org/v/v10/working-with-r/r-documentation/introduction-ospsuite#installing-the-r-toolbox)
* [here](https://docs.open-systems-pharmacology.org/v/v10/#working-with-r)

**Comment by StephanSchaller** [comment]
@PavelBal, can you? ;-)

**Comment by PavelBal** [comment]
Yepp already on it.

**Pull request merged**